PoTMod.cs

```diff
@@ -44,6 +44,7 @@ public override void Load()
 		Debug.Assert(Name == ModName, "Internal mod name does not match expected constant.");
 
 		ChatManager.Register<ClassNounTagHandler>("plrclass");
+		ChatManager.Register<TextureTagHandler>("tex");
 	}
```

Comment:
As in the other PR, load via a nested `ILoadable`. Doing that for ClassNoun too would be nice as well.